File: src/test/java/ru/anarcom/octopus/utilus/ResourceReader.java
package ru.anarcom.octopus.utilus;

import java.io.IOException;
import java.io.InputStream;
import java.nio.charset.StandardCharsets;

public class ResourceReader {
    public static String getResource(String path) throws IOException {
        if (path.indexOf('/') == 0) {
            path = path.substring(1);
        }
        InputStream is = ClassLoader.getSystemResourceAsStream(path);
        return new String(is.readAllBytes(), StandardCharsets.UTF_8);
    }
}

File: src/test/kotlin/ru/anarcom/octopus/utilus/ResourceReader.kt
package ru.anarcom.octopus.utilus

import java.io.IOException
import java.nio.charset.StandardCharsets


class ResourceReader {
companion object {
    @kotlin.jvm.Throws(IOException::class)
    fun getResource(path: String): String {
        var path = path
        if (path.indexOf('/') == 0) {
            path = path.substring(1)
        }
        val tmp = ClassLoader.getSystemResourceAsStream(path)
        return String(tmp.readAllBytes(), StandardCharsets.UTF_8)
    }
}
}
